### DISCUSSION TITLE
Carlinkit comparability

### DISCUSSION THREAD

**Discussion by nrushforth** [tesla-android/issue-tracker#208] [General]
Hi

I have the option to purchase CPC200-CP2A at a reduced price. Does anyone know if this would work with Tesla Android?

thanks.

**Comment by mikegapinski** [comment]
Sorry, it won't work

**Reply by nrushforth** [reply]
Wow fast reply. 

So is it only the 2 you mention in the hardware requirements?

**Reply by mikegapinski** [reply]
Yup, and the CPC200-AutoKit is out of production I guess, the CCPA is an updated model. It doesn't matter for Tesla Android, buy a cheaper one

**Reply by DrewHager571** [reply]
Yep. CPC200-AutoKit is out of production. I guess the new version CCPA will be better than CPC200-AutoKit. Cuz CCPA support both wireless CP and AA.

**Comment by nrushforth** [comment]
Thanks Michael. Do you have any suggested ones to buy and from where. Might be useful to include these in a FAQ.

**Reply by DrewHager571** [reply]
I have an 20% discount code "New2023" if you need it.

https://autokitcarplay.com/products/autokit-carplay-adapter-for-android-car-stereo

**Reply by nrushforth** [reply]
Thanks for the offer but I got a CCPA from Amazon UK for £50.

**Reply by mikegapinski** [reply]
@nrushforth someone created this account just to share this offer link here...

**Reply by philippefraser** [reply]
Hi @nrushforth im in UK too and a bit lost with hardware. Can you share any links for the CCPA and raspberry and case? If it’s not too much trouble.

**Reply by nrushforth** [reply]
Hi. 

I got this from Amazon
https://amzn.eu/d/bZlgdF8

I already had a Raspberry Pi 4 4Gb. I bought a 128GB micro sd card from Amazon for it. I bought an integral branded one and it has been ok. As to a case I just bought the Genuine Raspberry Pi case and placed heatsinks on the chips to help cool them, but to be honest it has been ok. Finally I power it with a short USBC to USBC cable that I plug into the USBC socket in the centre console. 

Hope this helps.

**Reply by philippefraser** [reply]
That’s very helpful. Thank you

**Reply by adamChrystie** [reply]
> Hi @nrushforth im in UK too and a bit lost with hardware. Can you share any links for the CCPA and raspberry and case? If it’s not too much trouble.

Here is [the main page to a blog I maintain about Tesla Android](https://atomfx.com/adamchryTechBlog/tesla-android-landing-page/). It targets the CarPlay user mostly but still good info for all new people. I outline two setups I've made. list the parts and explain why I use them. The intent was to help non technical people be less intimidated. I get no money by people visiting the blog though my kids say I should setup advertisement banners & make a YouTube channel ..lol.

**Comment by miaszh** [comment]
Hi, I'm new here ..
Can anyone confirm this works fine:
https://amzn.eu/d/coCZFs1

Thanks in advance

**Comment by adamChrystie** [comment]
Yes. That is one of the recommended items which the website mentions & is
what most people use.

**Comment by miaszh** [comment]
Hi
I've finally got the cpc200-CCPA, unfortunately, when I connect it, the rPI doesn't broadcast the "TeslaAndroid-XXX" SSID anymore! So I can't actually access the network ..
Also, it seems to be stuck..!
Anyone with the same model could help with a work around?

**Reply by mikegapinski** [reply]
It seems like an obvious power delivery issue. Make sure the USB-C->USB-C cable is decent, and the power source has at least 3.0A over USB-C Power Delivery.

If your car is a 2021+ the ports in the center console are sufficient
